### DISCUSSION TITLE
Start_server.bat error, python no module named 'encodings'

### DISCUSSION THREAD

**Discussion by wnzzzj** [kbengine/kbengine#1523] [General]
Windows7  to start start_server.bat Error  "Fatal Python error:initfsencoding: unable to load the file system codec"
"ModuleNotFoundError: No module named 'encoding'",  Python version is 3.8.8,  Has anyone found this kind of error?

**Comment by huhubis** [comment]
环境没设置好吧，  python包都没找到

**Comment by riyueguanghui** [comment]
> Windows7 to start start_server.bat Error "Fatal Python error:initfsencoding: unable to load the file system codec" "ModuleNotFoundError: No module named 'encoding'", Python version is 3.8.8, Has anyone found this kind of error?

最后是怎么解决的，我也遇到了，有毒

**Comment by ve-i-uj** [comment]
KBEngine uses Python built into the engine (not Python installed in the system). As one of the options, you can try to remove the python from the operating system (if this is not critical) and rebuild the engine.

**Comment by riyueguanghui** [comment]
已收到邮寄，谢谢！
